build.gradle

```diff
@@ -49,23 +49,6 @@ ext {
   shouldUseDelombok = project.hasProperty('useDelombok')
 }
 
-task resolveDependencies {
```

Comment:
Was failing on https://github.com/kelloggm/checkerframework-gradle-plugin/issues/109 for some time